src/friendly-iframe-embed.js

```diff
@@ -381,6 +382,7 @@ export class FriendlyIframeEmbed {
     this.removeResources_();
     disposeServicesForEmbed(this.win);
     if (this.ampdoc) {
+      disposeServicesForDoc(this.ampdoc);
```

Comment:
Should this just be part of the `dispose()` call?

Reply:
Yes. Of course. Thanks for point it out.

Reply:
Done